.rubocop.yml

```diff
@@ -99,6 +103,7 @@ Metrics/BlockLength:
     - 'app/controllers/v0/apidocs_controller.rb'
 
 # Don't worry about ambiguous blocks in RSpec
+# Still recommended to disable specs in accordance to this lint rule 8/2019
```

Comment:
can you take a look through this PR and make sure that all changes are yours?  for example, this comment came from @LindseySaari's branch, right?